HealthTracker/src/data/mappers/bloodPressureMapper.ts

```diff
@@ -0,0 +1,28 @@
+import { BloodPressureRecordModel } from '../models/bloodPressureRecordModel';
+import {BloodPressureReading} from '../../domain/models/bloodPressureReading'
+import { format } from 'date-fns';
+
+export const toRecordModel = (formData: BloodPressureReading): BloodPressureRecordModel => {
+
+  const sysNum = parseInt(formData.systolic ?? "", 10);
```

Comment:
Use Number() instead of parseInt, Number strictly, and if any value is a number throw and exemption